### PR TITLE
Logs

### PR DESCRIPTION
**Pull request by ShamaUgale**
Log4j logs implementation

### CODE REVIEW

File: src/main/java/com/znsio/e2e/runner/Runner.java [hunk 1]
```diff
@@ -88,27 +91,32 @@
     private static Map<String, Map> testDataForEnvironment;
     private static Map applitoolsConfiguration = new HashMap();
     private final Properties properties;
+    private final String DEFAULT_LOG_PROPERTIES_FILE = "src/main/resources/log4j.properties";
```
Comment (anandbagmar) [inline src/main/java/com/znsio/e2e/runner/Runner.java:94]: Do we need this file name to be ./src/.....

File: src/main/java/com/znsio/e2e/runner/Runner.java [hunk 2]
```diff
@@ -171,10 +179,10 @@ public static String getTestData (String key) {
...
-        System.out.println("unified-e2e Runner");
-        System.out.println("Provided parameters:");
+        LOGGER.info("unified-e2e Runner");
+        LOGGER.info("Provided parameters:");
         for (int i = 0; i < args.length; i++) {
-            System.out.println("\t" + args[i]);
+            LOGGER.info("\t" + args[i]);
         }
         if (args.length != 3) {
             throw new InvalidTestDataException("Expected following parameters: 'String configFilePath, String stepDefDirName, String featuresDirName, String logDirName");
```
Comment (anandbagmar) [inline src/main/java/com/znsio/e2e/runner/Runner.java:188]: Can you remove the earlier incorrect checking - String logDirName?

File: src/test/resources/com/znsio/e2e/features/android/configs/config.properties [hunk 3]
```diff
@@ -10,6 +10,7 @@ BROWSER=chrome
 ENVIRONMENT_CONFIG_FILE=./src/test/resources/environments.json
 IS_VISUAL=false
 LOG_DIR=target
+LOG_PROPERTIES_FILE= ./src/test/resources/log4j.properties
```
Comment (anandbagmar) [inline src/test/resources/com/znsio/e2e/features/android/configs/config.properties:13]: Update the same in mobilab_config.properties as well

**Review by anandbagmar — COMMENTED** [review]
Comments added in PR.

Also, 
1. Update the README
2. At the end of the test, upload generated log file to ReportPortal (assuming log file is generated per scenario)

File: src/main/java/com/znsio/e2e/runner/Runner.java (hunk 1 above)
Comment (ShamaUgale) [reply src/main/java/com/znsio/e2e/runner/Runner.java:94]: not required. Works without it!

File: src/main/java/com/znsio/e2e/runner/Runner.java (hunk 2 above)
Comment (ShamaUgale) [reply src/main/java/com/znsio/e2e/runner/Runner.java:188]: its already changed.

File: src/test/resources/com/znsio/e2e/features/android/configs/config.properties (hunk 3 above)
Comment (ShamaUgale) [reply src/test/resources/com/znsio/e2e/features/android/configs/config.properties:13]: ok

**Pull request merged**